boards/nxp/mimxrt1170_evk/mimxrt1170_evk_mimxrt1176_cm7_B.yaml

```diff
@@ -23,8 +23,10 @@ supported:
   - hwinfo
   - i2c
   - mipi_dsi
+  - netif:eth
+  - pwm
   - spi
-  - usb_device
+  - usbd
```

Comment:
so we only support new usb stack?

Reply:
hmmm... not sure if I intended to deprecate the older stack... I know @jfischer-no is moving towards that direction. This might have been a fat finger on my part.

Reply:
It is only a test feature and does not reflect what is not supported. It is up to you where Twister picks up your board. Probably `usb_device` will be removed after the commit release.